screens/HomeScreen.tsx

```diff
@@ -141,8 +144,11 @@ const HomeScreen = ({
   updateSpotPrice,
   updateTokensMeta,
   updateTransactions,
-  updateUtxos
+  updateUtxos,
+  tokenBlacklist
 }: Props) => {
+  const [isShowingBlacklist, setIsRevealed] = useState(false);
```

Comment:
Please follow the useState naming convention I outlined in last review, where ther setter is based on the variable's name
So in this case it would be 
`const [isShowingBlacklist, setIsShowingBlacklist] = useState(false);`